src/coreclr/vm/callhelpers.cpp

```diff
@@ -534,7 +534,13 @@ void MethodDescCallSite::CallTargetWorker(const ARG_SLOT *pArguments, ARG_SLOT *
 #ifdef CALLDESCR_REGTYPEMAP
     callDescrData.dwRegTypeMap = dwRegTypeMap;
 #endif
+#if defined(TARGET_RISCV64) || defined(TARGET_LOONGARCH64)
+    // Temporary conversion to old flags, CallDescrWorker needs to be overhauled anyway
```

Comment:
Why would `CallDescrWorker` need to be overhauled?

`CallDescrWorker` receives and returns the raw values of registers and stack. It is up to the `CallDescrWorker` caller (e.g this method) to convert the managed arguments to register and stack values and back.